sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -711,10 +722,21 @@ def update_message(self,
         if receipt is None:
             raise ValueError("pop_receipt must be present")
         if message_text is not None:
-            self._config.message_encode_policy.configure(
-                self.require_encryption,
-                self.key_encryption_key,
-                self.key_resolver_function)
+            try:
+                self._config.message_encode_policy.configure(
+                    self.require_encryption,
+                    self.key_encryption_key,
+                    self.key_resolver_function,
+                    encryption_version=self.encryption_version)
+            except TypeError:
+                warnings.warn(
+                    "message_encode_policy.configure does not accept encryption_version parameter.",
```

Comment:
Why is this warning different to the one above?

Reply:
Oops, I had updated this at one point and missed the second one. Will update.